### PR TITLE
Create accessibility-how-to.md

### PR DESCRIPTION
**Pull request by hhartingrex**
@mgifford Here is a starting place. Once we get this page working - first draft - I will adjust the left navigation bar. Thank you!

----
:books: Documentation preview :books:: https://civicactions-handbook--1086.org.readthedocs.build/en/1086/

### CODE REVIEW

**Comment by hhartingrex** [comment]
Looks like I need to get the menu change in for this PR to be accepted

**Review by dmundra — COMMENTED** [review]
@hhartingrex do you want to deploy a work in progress page or hold off till it has more content in it?

**Comment by hhartingrex** [comment]
Thank you @dmundra . 
@mgifford is our handoff. He is going to take this and spiff it up, then we will publish first draft. Thanks Mike!

File: practice-areas/accessibility-how-to.md
Comment (mgifford) [inline practice-areas/accessibility-how-to.md]: @maggiewachs what do you think of this extension?

Comment (mgifford) [inline practice-areas/accessibility-how-to.md]: @maggiewachs what do you think of this extension?

Comment (mgifford) [inline practice-areas/accessibility-how-to.md]: @maggiewachs what do you think of this extension?

File: practice-areas/accessibility-how-to.md
Comment (maggiewachs) [inline practice-areas/accessibility-how-to.md]: This is admittedly a nit-picky comment :) But I like to reframe empathy as needing to learn from screen reader users' expertise. I'd change this to: "It is also important to observe and learn from the ways blind users navigate through the page. This video demonstrates [How A Screen Reader User Surfs The Web](https://www.youtube.com/watch?v=OUDV1gqs9GA)."

File: practice-areas/accessibility-how-to.md
Comment (maggiewachs) [inline practice-areas/accessibility-how-to.md]: Excellent point!

**Review by maggiewachs — COMMENTED** [review]
This is great, ship it!

File: practice-areas/accessibility-how-to.md
Comment (dmundra) [inline practice-areas/accessibility-how-to.md]: @mgifford looks like this link `\[Keyboard-only testing\](https://webaim.org/techniques/keyboard/)` is being escaped.

**Review by dmundra — CHANGES_REQUESTED** [review]
@mgifford looking good, one minor link fix and then this can be merged.

**Comment by ContentBrewmaster** [comment]
Before we merge this, let's consider a couple of things:
- we now have 2 pages for the Accessibility practice area, so they need their own folder
- the steps should be change to H3 so they show up in the "on this page" nav on the right

**Comment by hhartingrex** [comment]
@ContentBrewmaster Can you do the folder for us? I can change the page to have the H3s. Thank you!

File: practice-areas/accessibility-how-to.md
Comment (mgifford) [reply practice-areas/accessibility-how-to.md]: I think I squashed that now.

**Comment by hhartingrex** [comment]
@dmundra Are there any more requested changes that we have not met? I think we are ready! THANK YOU for help (as usual!)

**Comment by dmundra** [comment]
> @dmundra Are there any more requested changes that we have not met? I think we are ready! THANK YOU for help (as usual!)

@hhartingrex can you take a look at #1092. It will have the folder changes requested for this branch.

**Review by ContentBrewmaster — COMMENTED** [review]
@dmundra @maggiewachs there seem to be unresolved comments in this content. Can you take a look?

**Review by dmundra — COMMENTED** [review]
@hhartingrex, @mgifford FYI I merged my folder changes for the accessibility documents into this PR.

**Review by dmundra — APPROVED** [review]
Changes look good to me! @mgifford can you review the comment from Maggie?

File: practice-areas/accessibility-how-to.md
Comment (mgifford) [reply practice-areas/accessibility-how-to.md]: That makes sense to me. Like that you include a link to a Smashing video. So just replace that sentence with line 25 or just part of it?

Comment (ContentBrewmaster) [reply practice-areas/accessibility-how-to.md]: @maggiewachs can you follow up on Mike's suggestion so we can merge this PR?

**Comment by ContentBrewmaster** [comment]
@mgifford can you look at Maggie's comment so we can merge this PR?

File: practice-areas/accessibility-how-to.md
Comment (mgifford) [reply practice-areas/accessibility-how-to.md]: Ok, I've modified it. Now reads as:  
  
`It is very difficult for sighted users to understand how a blind person uses a screen reader. You have to essentially forget the visual layout and learn how to navigate the page content with headers, links and aria-regions. It is also important to observe and learn from the ways blind users navigate through the page. This video demonstrates [How A Screen Reader User Surfs The Web](https://www.youtube.com/watch?v=OUDV1gqs9GA).`

**Comment by dmundra** [comment]
@maggiewachs, @hhartingrex, @mgifford we merged these changes since we have a new feature coming down soon (#1123). Once that is merged, feel free to make adjustments to changes introduced here.

**Pull request merged**
